compiler/ztests/pushdown-index.yaml

```diff
@@ -1,47 +0,0 @@
-script: |
-  zc -C -O "from 'pool-ts'| x=='hello' or x==1.0"
-  echo ===
-  zc -C -O "from 'pool-ts'| x > 1 y <= 1.0"
-  echo ===
-  zc -C -O "from 'pool-ts'| x=='hello' or x!=1.0"
-  echo ===
-  zc -C -O "from 'pool-ts'| x=='hello' or !(y==2 or y==3)"
```

Comment:
I think this is our only coverage of `zc -C -O` output that includes pushdowns, so maybe keep it (with the necessary changes to the output) and rename to `pushdown.yaml`?